requirements/constraints.txt

```diff
@@ -130,3 +130,7 @@ numpy<2.0.0
 # Two lines were added in 1.14.4 that make file_exists_in_storage function always return False,
 # as the default value of AWS_S3_FILE_OVERWRITE is True
 django-storages<1.14.4
+
+# social-auth-app-django 5.4.2 introduces a new migration that will not play nicely with large installations.
```

Comment:
Can you add some specificity here? Is it that the target table size is huge and will result in a really long migration run?